pkg/transforms/common.go

```diff
@@ -156,6 +156,47 @@ func CommonEdges(uid string, ns NodeStore) []Edge {
 
 	// deployer subscriber edges
 	ret = append(ret, edgesByDeployerSubscriber(nodeInfo, ns)...)
+
+	// Kyverno
```

Comment:
I'm wondering if this would be cleaner as a separate function that might return an Edge, similar to the other `edgesBy...` functions.